en/restore-data-using-tidb-lightning.md

```diff
@@ -272,24 +293,89 @@ The method of deploying TiDB Lightning varies with different methods of granting
+```shell
+helm delete ${release_name} --purge
+```
 
-1. Delete the lightning job by running `kubectl delete job -n ${namespace} ${release_name}-tidb-lightning`.
+To destroy tidb-lightning, execute the following command:
 
-2. Create the lightning job again with `failFast` disabled by `helm template pingcap/tidb-lightning --name ${release_name} --set failFast=false -f tidb-lightning-values.yaml | kubectl apply -n ${namespace} -f -`.
+{{< copyable "shell-regular" >}}
 
-3. When the lightning pod is running again, use `kubectl exec -it -n ${namespace} ${pod_name} sh` to `exec` into the lightning container.
+```shell
+helm delete ${release_name} --purge
+```
 
-4. Get the startup script by running `cat /proc/1/cmdline`.
+## Troubleshoot TiDB Lightning
 
-5. Diagnose the lightning following the [troubleshooting guide](https://pingcap.com/docs/stable/troubleshoot-tidb-lightning/).
+When TiDB Lightning fails to restore data, you cannot simply restart it. **Manual intervention** is required. Therefore, the TiDB Lightning's `Job` restart policy is set to `Never`.
 
-## Destroy TiKV Importer and TiDB Lightning
+> **Note:**
+>
+> If you have not configured to persist the checkpoint information in the target TiDB cluster, other MySQL-compatible databases or a shared storage directory, after the restore failure, you need to first delete the part of data already restored to the target cluster. After that, deploy tidb-lightning again and retry the data restore.
+
+If the lightning fails to restore data, and if you have configured to persist the checkpoint information in the target TiDB cluster, other MySQL-compatible databases or a shared storage directory, follow the steps below to do manual intervention:
+
+1. View the log by executing the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl logs -n ${namespace} ${pod_name}
+    ```
+
+    * If you restore data using the remote data source, and the error occurs when TiDB Lightning downloads data from remote storage:
+
+        1. Address the problem according to the log.
+        2. Deploy tidb-lightning again and retry the data restore.
+
+    * For other cases, refer to the following steps.
+
+2. Refer to [TiDB Lightning Troubleshooting](https://docs.pingcap.com/tidb/stable/troubleshoot-tidb-lightning) and learn the solutions to different issues.
+
+3. Address the issues accordingly:
+
+    - If `tidb-lightning-ctl` is required:
+
+        1. Configure `dataSource` in `values.yaml`. Make sure the new `Job` uses the data source and checkpoint information of the failed `Job`:
+
+            - In the local or ad hoc mode, you do not need to modify `dataSource`.
+            - In the remote mode, modify `dataSource` to the ad hoc mode. `dataSource.adhoc.pvcName` is the PVC name created by the original Helm chart. `dataSource.adhoc.backupName` is the backup name of the data to be backed up.
```

Comment:
`to be backed up` --> `have been backed up`? if so, ditto for L377

Reply:
`to be restored`. Fixed in d2ba2f6.